src/main/java/com/example/springbootapp/exception/EmployeeExceptionHandler.java

```diff
@@ -8,21 +10,33 @@
 @RestControllerAdvice
 public class EmployeeExceptionHandler {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(EmployeeExceptionHandler.class);
+
     @ExceptionHandler
     @ResponseStatus(HttpStatus.NOT_FOUND)
     public ResponseError employeeDoesNotExist(EmployeeNotFoundException exception){
+        LOGGER.error(exception.getMessage(), exception);
         return new ResponseError(HttpStatus.NOT_FOUND, exception.getMessage());
     }
 
+    @ExceptionHandler
+    @ResponseStatus(HttpStatus.CONFLICT)
+    public ResponseError employeeAlreadyExists(EmployeeNotUniqueException exception){
```

Comment:
Please check formatting for the whole project

Reply:
Formatting is checked in whole project.